<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerWindow</class>
 <widget class="QMainWindow" name="registerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>-40</y>
      <width>671</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{
	border-image: url(:/IMG_2027.JPG);}
</string>
    </property>
    <widget class="QLineEdit" name="usernameLineEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(244,244,244,2);
font: 12pt &quot;黑体&quot;;
background-color: rgb(145, 145, 145,120);
	border-radius:3px;
	color:rgb(0,0,0);
border:1px solid #e1e0e0;
border-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	bcakground-color: rgb(255,132,139,0);
font: 12pt &quot;04b_21&quot;;
	border-radius:3px;
	color:rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>190</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 12pt &quot;黑体&quot;;
background-color: rgb(145, 145, 145,120);
	border-radius:3px;
	color:rgb(0,0,0);
border:1px solid #e1e0e0;
border-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	bcakground-color: rgb(255,132,139,0);
font: 12pt &quot;04b_21&quot;;
	border-radius:3px;
	color:rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="telphoneLineEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 145, 145,120);
	border-radius:3px;
	color:rgb(0,0,0);
font: 12pt &quot;黑体&quot;;
border:1px solid #e1e0e0;
border-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>250</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	bcakground-color: rgb(255,132,139,0);
font: 12pt &quot;04b_21&quot;;
	border-radius:3px;
	color:rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IDLineEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>310</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 145, 145,120);
	border-radius:3px;
	color:rgb(0,0,0);
font: 12pt &quot;黑体&quot;;
border:1px solid #e1e0e0;
border-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>310</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	bcakground-color: rgb(255,132,139,0);
font: 12pt &quot;04b_21&quot;;
	border-radius:3px;
	color:rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>身份证号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RealNameLineEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>370</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 145, 145,120);
	border-radius:3px;
	color:rgb(0,0,0);
font: 12pt &quot;黑体&quot;;
border:1px solid #e1e0e0;
border-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>370</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	bcakground-color: rgb(255,132,139,0);
font: 12pt &quot;04b_21&quot;;
	border-radius:3px;
	color:rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>真实姓名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registerPushButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>440</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Chh5.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>540</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;Arial&quot;;
color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string>点我注册哟！</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
